Preferences.sublime-settings

```diff
@@ -4,6 +4,11 @@
 	"kickass_startup_file_path": "Startup",
 	"kickass_empty_bin_folder_before_build" : "true",
 	"kickass_breakpoint_filename" : "breakpoints.txt",
+	"kickass_debug_logfile_filename" : "${kickass_output_path}/${build_file_base_name}_BuildLog.txt",
+	"kickass_run_logfile_filename" : "${kickass_output_path}/${build_file_base_name}_BuildLog.txt",
```

Comment:
Value for `kickass_debug_logfile_filename` and `kickass_run_logfile_filename` should differ from value for `kickass_compile_logfile_filename`, since the last is output from kick assembler, while the other 2 are output from Vice.